pallets/issue/Cargo.toml

```diff
@@ -55,6 +55,8 @@ oracle = { path = "../oracle", features = ['testing-utils'] }
 orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v0.9.33" }
 orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v0.9.33" }
 
+dia-oracle = { git = "https://github.com/TorstenStueber/oracle-pallet", branch = "master", default-features = false }
```

Comment:
(As we learned in todays sync meeting) this can be changed to the 'real' repo now.

Reply:
Done